.github/CODEOWNERS

```diff
@@ -0,0 +1,23 @@
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+#*       @global-owner1 @global-owner2
+*       @terry-klaytn @neoofklaytn @toniya-klaytn @chloe-klatyn
```

Comment:
the codeowner should be updated and could consider including some of our core dev leads thanks

Reply:
added and pushed.